p/themes/Alternative-Dark/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Alternative Dark",
+  "author": "Ghost",
```

Comment:
@fabianski7
is there a reason why you chose "Ghost" as author name and not your username?